assets/styles/fonts.scss

```diff
@@ -0,0 +1,17 @@
+@font-face {
+    font-family: "Sara";
+    font-weight: 400;
+    src: url("./../fonts/Sara_Regular.woff");
+}
+
+@font-face {
+    font-family: "Sara";
+    font-weight: 600;
+    src: url("./../fonts/Sara_Semibold.woff");
+}
+
+@font-face {
+    font-family: "Sara";
+    font-weight: 700;
+    src: url("./../fonts/Sara_Bold.woff");
```

Comment:
Kullekin font-facelle voisi lisätä ´font-display: swap;`, joka on suositeltu FOUT:in minimoimiseksi.

Reply:
Hyvä huomio, lisätty! 👍